lib/logstash/outputs/mongodb.rb

```diff
@@ -38,20 +41,18 @@ def register
     @db = conn.use(@database)
   end # def register
 
-  public
   def receive(event)
-    
-
     begin
-      if @isodate
-        # the mongodb driver wants time values as a ruby Time object.
-        # set the @timestamp value of the document to a ruby Time object, then.
-        document = event.to_hash
-      else
-        document = event.to_hash.merge("@timestamp" => event["@timestamp"].to_json)
+      # Our timestamp object now has a to_bson method, using it here
+      # why not use to_hash_with_metadata?
```

Comment:
I don't follow this comment? I guess if we change the to_bson, not to include metadata, we can fairly use it here. what do you think?